fuel/transformers/__init__.py

```diff
@@ -271,12 +395,17 @@ class Cache(Transformer):
         refilled when needed through the :meth:`get_data` method.
 
     """
-    def __init__(self, data_stream, iteration_scheme):
+    def __init__(self, data_stream, iteration_scheme, **kwargs):
```

Comment:
It's requiring an instance of `BatchSizeScheme` as its iteration scheme, so it means `produces_examples=False` is implicitly forced through that restriction. I'll add a comment making this explicit.